src/test/http-mocks/fees.ts

```diff
@@ -2,89 +2,157 @@ import * as config from 'config'
 import * as mock from 'nock'
 import * as HttpStatus from 'http-status-codes'
 
-const serviceBaseURL: string = `${config.get('fees.url')}/range-groups`
-
-const issueFeeCode: string = config.get<string>('fees.issueFee.code')
-const hearingFeeCode: string = config.get<string>('fees.hearingFee.code')
-
-const calculationOutcome = {
-  amount: 2500,
-  fee: {
-    code: 'X0048',
-    description: 'Civil Court fees - Hearing fees - Claim Amount - 0.01 up to 300 GBP',
-    amount: 2500,
-    type: 'fixed'
-  }
+const service: string = (config.get<string>('fees.service'))
```

Comment:
same comment as above about the types,
consider creating a fees config object and then you could just need to import one item
`const feeConfig = config.get<FeeConfig>('fees')`
REC

Also there's unneeded brackets around the config bit
REQ